backend/typescript/services/implementations/emailService.ts

```diff
@@ -166,12 +189,14 @@ class EmailService implements IEmailService {
       of the fields, reach out to camps@focusonnature.ca. <br>
       <ul>
         <li><b>Camp name:</b> ${camp.name}</li>
-        <li><b>Camp location:</b> ${camp.location}</li>
-        <li><b>Session dates:</b> ${sessionDatesToString(
-          campSession.dates,
-        )}</li>
+        <li><b>Camp location:</b> ${campLocationString}</li>
+        <li><b>Session dates:</b></li>
+        <ol>
+          ${sessionDatesListItems.join("")}
```

Comment:
Think that dates should be separated by commas `.join(", ")`

Reply:
this join is actually used to render an array of list item strings to the html template. The change to separate the dates with commas should be in the `getSessionDatesListItems` function. I changed it so the email looks like this (ignore the color of the text- that's my gmail) 

<img width="399" alt="image" src="https://user-images.githubusercontent.com/57159970/209758555-029cd9e1-76eb-4507-be7c-6a4f30107179.png">